### DISCUSSION TITLE
This is such a great provider for TF and Proxmox

### DISCUSSION THREAD

**Discussion by bartei** [bpg/terraform-provider-proxmox#201] [General]
Thanks for putting this provider together, I don't know why this is not more popular than other for terraform! 

I'd love to help you testing your provider if you need help :)

**Comment by bpg-dev** [comment]
Hey @bartei, thanks, that would be great!

I can really use some help in pin-pointing issue(s) in a scenario described in #186.
From all my testing it seems to be a bottleneck in the PVE itself. Ideally I'd like to reproduce the same problem but without TF, i.e. using PVE CLI and some scripting. Just to see if eliminating the provider from the equation improves the things.

Would you be able to tackle that?
